### PR TITLE
Update to new theme

### PR DESCRIPTION
**Pull request by bmorcos**
**Motivation and context:**

Update to new sphinx theme

**How long should this take to review?**

- Quick (less than 40 lines changed or changes are straightforward)

**Where should a reviewer start?**

Build the docs and take a look

**Types of changes:**

- Non-code change (touches things like tests, documentation, build scripts)

**Checklist:**

- [x] I have read the **CONTRIBUTING.rst** document.
- [x] I have updated the documentation accordingly.
- [x] I have included a changelog entry.
- N/A I have tested this with all supported devices.
- N/A I have added tests to cover my changes.
- N/A I have run the test suite locally and all tests passed.

### CODE REVIEW

File: docs/examples.rst [hunk 1]
```diff
@@ -205,30 +205,30 @@ controlled by a standard PD controller. It can be seen that in this case,
...
+.. list-table::
+  :widths: 50 50
+  :align: center
 
-      * -
+  * -
 
-         .. do not delete this table row. It adds the line above the table
-            to match the one added by the row below.
+     .. do not delete this table row. It adds the line above the table
```
Comment (xchoo) [inline docs/examples.rst]: I think we can remove this... the new theme seems to add a line above the table (at least, for me it did)

File: docs/examples.rst [hunk 2]
```diff
@@ -312,45 +312,45 @@ When the simulation is running, the agent should move around the world, as
...
+.. list-table::
+  :widths: 33 33 33
+  :align: center
 
-      * -
+  * -
 
-         .. do not delete this table row. It adds the line above the table
-            to match the one added by the row below.
+     .. do not delete this table row. It adds the line above the table
```
Comment (xchoo) [inline docs/examples.rst]: Same as above. remove?

File: docs/appendix.rst [hunk 3]
```diff
@@ -19,7 +19,7 @@ DE1-SoC
 
    .. image:: _static/de1.jpg
```
Comment (xchoo) [inline docs/appendix.rst]: modify to use PNG with transparent border

File: docs/getting_started.rst [hunk 4]
```diff
@@ -33,8 +33,8 @@ Installation Quick Reference Guide
 
          python nengo_fpga/id_extractor.py <board>
 
-   where **<board>** is the board name as it appears in ``fpga_config``.
-   See the `Copy Protection`_ section for more information.
+    where **<board>** is the board name as it appears in ``fpga_config``.
```
Comment (xchoo) [inline docs/getting_started.rst]: This rendered weird on my build. The one just below it rendered fine though... so maybe a tabbing issue?

Comment (xchoo) [reply docs/getting_started.rst]: image

File: docs/examples.rst (hunk 1 above)
Comment (xchoo) [reply docs/examples.rst]: This is what it looks like after I removed the first empty row. Double check that yours does the same thing too.

image

File: docs/getting_started.rst [hunk 5]
```diff
@@ -33,8 +33,8 @@ Installation Quick Reference Guide
 
          python nengo_fpga/id_extractor.py <board>
 
-    where **<board>** is the board name as it appears in ``fpga_config``.
-    See the `Copy Protection`_ section for more information.
+   where **<board>** is the board name as it appears in ``fpga_config``.
```
Comment (xchoo) [inline docs/getting_started.rst]: Should be tabbed over 1 more. It should be aligned to the i. and ii.

Comment (xchoo) [reply docs/getting_started.rst]: I dont know why github is saying this comment is on an outdated file...

Comment (bmorcos) [reply docs/getting_started.rst]: so close haha

**Pull request merged**